### PR TITLE
Update support page

### PR DESCRIPTION
**Pull request by fsansalvadore**
New Support page:
- Integrated CmdK (accessible from anywhere in www)
- Updated layout and links

Left FAQ/Troubleshooting out for now because it would require a bit more effort and thought.
Going to follow up in separate PR eventually.

Screenshot 2023-10-12 at 12 22 24

### CODE REVIEW

**Review by awalias — COMMENTED** [review]
1) `open a ticket` looks like the main CTA here, which seems to go against our quest to [reduce free tier tickets](https://www.notion.so/supabase/RFC-Reduce-Free-Tickets-64a1312536fb4bcebe7f1fb8f8b435a8?pvs=4)

maybe GH Discussions should be the main CTA (after AI docs search)

could make the GH discussions button white and change the copy to `Ask a question`

2) we should make it clear that `open a ticket` is for paying customers only. "Are you on a paid plan? -> Open a ticket"

File: packages/ui/src/components/Icon/icons/index.tsx [hunk 1]
```diff
@@ -86,6 +86,7 @@ export * as IconCrosshair from './IconCrosshair'
 export * as IconDatabase from './IconDatabase'
 export * as IconDelete from './IconDelete'
 export * as IconDisc from './IconDisc'
+export * as IconDiscussions from './IconDiscussions'
```
Comment (fsansalvadore) [inline packages/ui/src/components/Icon/icons/index.tsx:89]: @MildTomato it probably makes sense for us to extend this existing Icons component with custom ones so we can continue using a standardized api.

**Comment by KevinBrolly** [comment]
Amazing to see this @fsansalvadore 👍 

I think we could change/clarify the "Issues and Feature requests" section. In [the pricing page ](https://supabase.com/pricing) we tell customers to request features by using the "Feature Request" category of Github Discussions rather than making issues for feature requests Screenshot 2023-10-10 at 15 31 17

Maybe we could say more "Found a bug -> make issue" - "Got a feature request -> Make feature request in GH Discussions"

I would also love to improve the copy around "Response time might vary..." and even provide another page with examples of what to expect and more information but that can come in future PR. This is still big improvement 👍

File: apps/www/pages/support.tsx [hunk 2]
```diff
@@ -34,181 +62,151 @@ const Index = ({}: Props) => {
...
+              <div className="col-span-full lg:col-span-6 gap-2 flex flex-col">
+                <h2 className="text-xl lg:text-2xl tracking-tight">
+                  Can't find what you're looking for?
+                </h2>
+              </div>
+              <div className="col-span-full lg:col-start-7 lg:col-span-6 flex flex-col gap-4">
+                <div className="flex flex-col">
+                  <p className="text-light">The Supabase Support Team is ready to help.</p>
+                  <p className="text-lighter text-sm">
+                    Response time might vary depending on plan type and severity of the issue.
```
Comment (fsansalvadore) [inline apps/www/pages/support.tsx:190]: @awalias @KevinBrolly the purpose of this text is to discourage users on free plans to open too many tickets.
Wdyt? Would you phrase it differently or do you guys think it conveys that message well enough?

Open to suggestions ✌️

**Comment by fsansalvadore** [comment]
> Maybe we could say more "Found a bug -> make issue" - "Got a feature request -> Make feature request in GH Discussions"

@KevinBrolly So you would maybe have two separate cards for that? Or two links in the first card?
I'll try something out 👍

**Comment by KevinBrolly** [comment]
Could be two seperate cards or two seperate buttons. I guess the important part is just that its two seperate CTA - if bug please make an issue so we can fix it, if feature request please use "proper" feature request channel via GH Discussions so it gets prioritised/voted on accordingly.

**Review by awalias — APPROVED** [review]

**Pull request merged**